packages/core/react-loosely-lazy/__tests__/integration.test.tsx

```diff
@@ -131,7 +131,7 @@ describe('hydrates', () => {
     describe('when ssr is false', () => {
       const ssr = false;
 
-      it('by rendering the server fallback, before rendering the fallback and replacing', async () => {
+      it('by rendering  and persisting the server fallback, before replacing', async () => {
```

Comment:
Change in behaviour: in case we render the fallback on SSR, we no longer hydrate it immediately (with a live fallback) as that is not how real Suspense works. So we better align